sam_handler/handler.py

```diff
@@ -245,7 +245,6 @@ def _package(self, invoker: SamInvoker):
             'region': self.connection_manager.region,
             's3-prefix': self.artifact_key_prefix,
             'output-template-file': self.destination_template_path,
-            'template-file': str(self.sam_template_path.absolute())
```

Comment:
As I explained in the PR description, this is the wrong template. We need to let `sam package` pick up the right template on its own.